client/db/Cargo.toml

```diff
@@ -47,3 +47,6 @@ tempfile = "3"
 [features]
 default = []
 test-helpers = []
+with-kvdb-rocksdb = ["kvdb-rocksdb"]
+with-parity-db = ["parity-db"]
+with-subdb = []
```

Comment:
I like this `with-` naming scheme :^)

Reply:
Props to @gnunicorn for the naming convention :p